hs_access_control/models.py

```diff
@@ -2262,6 +2278,9 @@ def get_combined_privilege(self, this_user):
             raise HSAUsageException("Grantee is not a user")
         access_user = this_user.uaccess
 
+        if this_user.is_superuser:
+            return PrivilegeCodes.OWNER
+
```

Comment:
OK. I see now. Then this is not a problem. However, I still think it is better to move "access_user = this_user.uaccess" after "if this_user.is_superuser" check, since it is a wasteful statement for superuser case although it will not raise any errors.